meerk40t/gui/wxmscene.py

```diff
@@ -1292,10 +1292,16 @@ def on_close(self, event):
 
     @signal_listener("pipe;running")
     def on_driver_mode(self, origin, state):
+        new_color = None
         if state:
-            self.widget_scene.overrule_background = wx.RED
+            new_color = wx.RED
         else:
-            self.widget_scene.overrule_background = None
+            try:
```

Comment:
bit out of scope with the rest of the PR

Reply:
It was too simple to leave out...